vulture/core.py

```diff
@@ -46,6 +47,39 @@
 if sys.version_info < (3, 4):
     IGNORED_VARIABLE_NAMES |= {"True", "False"}
 
+ISSUE_CODE_MAP = {
+    "attribute": "V001",
+    "class": "V002",
+    "function": "V003",
+    "import": "V004",
+    "property": "V005",
+    "variable": "V006",
+    "unreachable_code": "V007",
+}
+
+NOQA_REGEXP = re.compile(
+    # We're looking for items that look like this:
+    # `# noqa`
+    # `#noqa`
+    # `#noqa: E123`
+    # `# noqa: E123`
+    # `# noqa: E123,W451,F921`
+    # `# noqa:E123,W451,F921`
+    # `# NoQA: E123,W451,F921`
+    # `# NOQA: E123,W451,F921`
+    # `# NOQA:E123,W451,F921`
+    # We do not want to capture the ``: `` that follows ``noqa``
+    # We do not care about the casing of ``noqa``
+    # We want a comma-separated list of errors
+    # https://regex101.com/r/4XUuax/3 full explenation of the regex
+    r"#( )?noqa(?::[\s]?(?P<codes>([A-Z]+[0-9]+(?:[,\s]+)?)+))?",
+    re.IGNORECASE,
```

Comment:
This expression was taken from https://gitlab.com/pycqa/flake8//blob/0ac337608b3d88079aa91bdcffed7b4df6fd28a8/src/flake8/defaults.py#L26 with an added case of `#noqa` (without a space b/w `#` and `noqa`) -- explanation URL updated.

Reply:
I think it makes sense to follow exactly what flake8 uses.

Reply:
Okay. Changed it back.